README.md

```diff
@@ -108,6 +135,40 @@ console.log("Thing status",thingStatusInfo.getStatus());
 
 See [openhab-js](https://openhab.github.io/openhab-js) for a complete list of functionality
 
+### Event Object
+
+**NOTE**: Note that `event` object is different in UI based rules and File based rules! This section is only valid for UI based rules.

```

Comment:
I would spell `File` in lowercase (`file`).
Maybe you could add a link to the documentation for the event in file-based rules.

Reply:
https://github.com/openhab/openhab-js/pull/110/commits/9315b918b593c1eca955ec1f6ddbc37e6843733e